plugins/inputs/postgresql_extensible/postgresql_extensible.go

```diff
@@ -221,7 +221,7 @@ func (p *Postgresql) accRow(meas_name string, row scanner, acc telegraf.Accumula
 		return err
 	}
 
-	if columnMap["datname"] != nil {
+	if *columnMap["datname"] != nil {
```

Comment:
I think this will crash if "datname" is not set, let's try to add a unit test for this function.